docs/serverless-v2/08-02-expose-lambda.md

```diff
@@ -69,3 +75,33 @@ Follows these steps:
     ```bash
     curl https://$NAME.$DOMAIN
     ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+> **NOTE:** The API Rules functionality that allow you to expose lambdas is an experimental feature, and it is not enabled by default in the Console UI. To use it, enable **Experimental functionalities** in the **General Settings** view before you follow the steps.
```

Comment:
You have three notes, 2 about Serverless in general (to create a Lambda and  Bind a service) and this one about API Rules. 
It can be different, I don't want to force any changes :D I'm just wondering why in this case it's not:

> **NOTE:** Serverless v2 is an experimental feature, and it is not enabled by default in the Console UI. To use its **API Rules [preview]** view, enable **Experimental functionalities** in the **General Settings** view before you follow the steps.

Reply:
I wrote it differently here because API Rules are not Serverless v2-related feature but an independent Kyma feature.